react/features/invite/components/AddPeopleDialog.web.js

```diff
@@ -221,6 +232,49 @@ class AddPeopleDialog extends Component {
     }
 
     /**
+     * Renders the error message if the add doesn't succeed.
+     *
+     * @returns {ReactElement|null}
+     * @private
+     */
+    _renderErrorMessage() {
+        if (!this.state.addToCallError) {
+            return null;
+        }
+
+        const { t } = this.props;
+        const supportString = t('inlineDialogFailure.supportMsg');
+        const supportLink = interfaceConfig.SUPPORT_URL;
+        const supportLinkContent
+            = ( // eslint-disable-line no-extra-parens
+                <span>
+                    <span>
+                        { supportString.padEnd(supportString.length + 1) }
+                    </span>
+                    <span>
+                        <a
```

Comment:
One issue here is if no support url is specified it will still look like there is a link to support. Clicking on it does nothing. Would it be possible to conditionally create the link?